climate-monitoring-client/src/main/java/climatemonitoring/CenterInfo.java

```diff
@@ -1,13 +1,60 @@
 package climatemonitoring;
 
+import climatemonitoring.core.Center;
+import climatemonitoring.core.ConnectionLostException;
+import climatemonitoring.core.DatabaseRequestException;
 import climatemonitoring.core.ViewState;
+import climatemonitoring.core.headless.Console;
 
 class CenterInfo extends ViewState {
 
 	@Override
 	public void onHeadlessRender(String args) {
 
-		throw new UnsupportedOperationException("Unimplemented method 'onHeadlessRender'");
+		try {
+
+			String[] splitargs = args.split(" ");
+			
+			String by = splitargs[0];
+			
+			if (by.equals("id")) {
+
+				String id = splitargs[1];
+				Center center = Handler.getProxyServer().getCenter(id);
+				
+				if (center != null) {
+
+					Console.write(center.toString());
+				} else {
```

Comment:
There is no `toString` method overridden for resource classes, so you have to print out manually every info about the center